IPs/RSKIP374.md

```diff
@@ -0,0 +1,50 @@
+---
+rskip: 374
+title: Reestablish the number of block confirmations for a PowPeg migration period
+created: 15-FEB-23
+author: MI
+purpose: Usa,Sec
+layer: Core
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |374           |
+| :------------ |:-------------|
+|**Title**      |Reestablish the number of block confirmations for a PowPeg migration period |
+|**Created**    |15-FEB-23 |
+|**Author**     |MI |
+|**Purpose**    |Usa,Sec |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a change to the number of block confirmations required for a PowPeg migration period.
+
+## Motivation
+
+As described in RSKIP357 [[1]](#references), as part of the efforts done to solve the peg-outs service incident[[2](#references)] it was necessary to increase the PowPeg migration period from 10,585 to 172,800 blocks to give enough time for a mining pool to process the non-standard transactions before the migration period finishes.
```

Comment:
This link references([[1]](#references), [[2](#references)]) don't take to any different place rather than the same page, because after the redirect to the link, the difference is almost imperceptible, seems like you stay on the same place.

What if we add the real reference? So it would look like this:

[1](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP357.md)

Reply:
Perhaps I could just remove the link then. And leave the reference as it is [1]. What do you think?